Add CLA and signoff check

viadea opened the pull request:
Add CLA and signoff check

viadea committed:
Add signoff check.

viadea committed:
Create CONTRIBUTING.md

Create CONTRIBUTING.md

viadea committed:
Add contributing guide

Add contributing guide

firestarman commented on .github/workflow/signoff-check.yml:
```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```
The year should be '2021'.

firestarman commented on .github/workflow/sigoff-check/Dockerfile:
```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```
The year should be '2021'.

firestarman commented on .github/workflow/sigoff-check/action.yml:
```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```
The year should be '2021'.

viadea replied:
@firestarman I just changed all 2020 to 2021. How do you think now? thx

firestarman commented on .github/workflow/sigoff-check/signoff-check:
```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020, NVIDIA CORPORATION.
```
The year should be '2021'.

sameerz commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,64 @@
+
+### Sign your work
+
+We require that all contributors sign-off on their commits. This certifies that the contribution is your original work, or you have rights to submit it under the same license, or a compatible license.
```
I suggest that lines be 100 characters or less for ease of readability

sameerz commented on CONTRIBUTING.md:
````diff
@@ -0,0 +1,64 @@
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+```
````
Add a note to describe where this signoff file was derived from

viadea replied:
@sameerz  Just added a link pointing to https://github.com/NVIDIA/spark-rapids . How do you think now?

sameerz replied:
Looks good

viadea committed:
Change 2020 to 2021

Change 2020 to 2021

viadea committed:
Change 2020 to 2021

Change 2020 to 2021

viadea committed:
Change 2020 to 2021

Change 2020 to 2021

viadea committed:
Change 2020 to 2021

Change 2020 to 2021

viadea committed:
Changed some words in Controbuting.md

Wrap some lines and add a note to point to https://github.com/NVIDIA/spark-rapids for this section.

firestarman commented:
LGTM

pxLi reviewed (approved):
LGTM
